tensorflow/compiler/mlir/tensorflow/transforms/tpu_cluster_formation.cc

```diff
@@ -344,8 +344,9 @@ LogicalResult ReplicateCluster(tf_device::ClusterOp cluster, int num_replicas) {
   for (auto& pos_and_input : llvm::enumerate(replicated_input_ops)) {
     auto input = pos_and_input.value();
     bool is_packed = llvm::cast<TF::TPUReplicatedInputOp>(input).is_packed();
+    const int input_getNumOperands = input->getNumOperands();
```

Comment:
Variable does not follow coding style regarding naming of local variables: the name should be in `snake_case`.